testing-common/src/main/java/io/opentelemetry/instrumentation/testing/util/TelemetryDataUtil.java

```diff
@@ -83,7 +83,12 @@ public static List<List<SpanData>> waitForTraces(
   public static void assertScopeVersion(List<List<SpanData>> traces) {
     for (List<SpanData> trace : traces) {
       for (SpanData span : trace) {
-        if (!span.getInstrumentationScopeInfo().getName().equals("test")) {
+        if (!span.getInstrumentationScopeInfo().getName().equals("test")
+            // opencensusshim is the underlying opentelemetry-java instrumentation which is
+            // automatically emitted upon use and which does not set a version at this time
+            && !span.getInstrumentationScopeInfo()
+                .getName()
+                .equals("io.opentelemetry.opencensusshim")) {
```

Comment:
@trask any thoughts on what should be done here? These spans are emitted by the opencensus-shim itself without a version attached.

Reply:
@dmarkwat can you update the opencensus-shim code to include a version?

Reply:
Yeah I can track that down in the shim, no problem.

Reply:
It's hard to tell with the chicken-and-egg between otel-java and otel-java-instrumentation, but I think this will solve the version? https://github.com/open-telemetry/opentelemetry-java/pull/5336

Reply:
@dmarkwat you should be able to rebase and pick up the change now that OTel SDK 1.25.0 has been published and `main` branch on this repo has been updated

Reply:
Rebased and looking happy :+1: